application/backend/src/main/resources/apps/typerefinery/components/widgets/ticker/styles/style.css

```diff
@@ -10,146 +10,181 @@
-  display: none;
-}
+.ticker-primaryTicker {
+    background: var(--primary-object-background-color);
+    min-width: 185px;
+    width: 100%;
+    height: 105px;
+    border-radius: 4px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    color: #495057;
+    box-shadow: 0 2px 1px -1px rgb(0 0 0 / 20%), 0 1px 1px 0 rgb(0 0 0 / 14%),
+      0 1px 3px 0 rgb(0 0 0 / 12%);
+    border-radius: 6px;
+    padding: 20px;
+    margin: 5px;
+    margin-bottom: 1%;
+  }
+  
+  .ticker-primaryTicker .container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    width: 100%;
+    height: 100%;
+  }
+  
+  .ticker-primaryTicker .body {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+  }
+  
+  .ticker-primaryTicker .content {
+    display: flex;
+    align-items: center;
+  }
+  
+  .ticker-primaryTicker .value {
+    font-weight: 500;
+    font-size: 24px;
+    display: flex;
+    align-items: center;
+    color: var(--primary-text-color);
+  }
+  
+  .ticker-primaryTicker .icon {
+    color: #00b1ff;
+  }
+  
+  .ticker-primaryTicker .title {
+    font-weight: 400;
+    font-size: 15px;
+    color: rgb(188, 188, 188);
+  }
+  
+  .ticker-primaryTicker .header {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+  }
+  
+  .ticker-primaryTicker .indicator {
+    margin-left: 10px;
+    font-size: 10px;
+    color: rgb(226 250 228);
+    flex-direction: row;
+    align-items: center;
+    display: flex;
+    border-radius: 4px;
+  }
+  
+  .ticker-primaryTicker .indicator_value {
+    margin-right: 10px;
+    margin-left: 10px;
+    font-size: 11px;
+  }
+  
+  .ticker-primaryTicker .indicator .icon {
+    font-size: 10px;
+    color: rgb(226 250 228);
+    margin-left: 10px;
+  }
+  
+  .ticker-primaryTicker .indicator .down {
+    color: #f55a32;
+  }
+  
+  .ticker-primaryTicker .indicator .same {
+    color: #a696d5;
+  }
+  
+  .indicator:has(.down) {
+    background: #4d3139;
+    color: #f55a32 !important;
+  }
+  
+  .indicator:has(.up) {
+    color: rgb(226 250 228) !important;
+    background: rgb(6, 42, 61);
+  }
+  
+  .ticker-primaryTicker .indicator .pi {
+    display: none;
+  }
+  
+  .indicator:has(.same) {
+    background: #33426a;
+    color: #a696d5 !important;
+  }
+  
+  .pi.pi-minus.same {
+    display: block;
+  }
+  
+  .pi.pi-arrow-up.up {
+    display: block;
+  }
+  .pi.pi-arrow-down.down {
+    display: block;
+  }
+  .ticker-primaryTicker .hours {
+    color: var(--primary-text-color);
+    font-size: 10px;
+    margin-left: 5px;
+  }
+  
+  .hours {
+    color: #FFF !important;
+  }
+  .title {
+    color: var(--subtitle-color) !important;
+  }
+  
+  .dn,
+  .path,
+  .refresh-time {
+    display: none;
+  }
+  /* secondary ticker */
+  .ticker-secondaryTicker {
```

Comment:
It will be more better If we have style inside the variants.

If anything is common then you can have them in ticker directory. 

for primary ticker variants/primary/styles/style.css

for secondary ticker variants/secondary/styles/style.css

Reply:
Implemented accordingly